appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -1793,13 +1793,31 @@
           360))
 
 (define (sin-degrees degrees)
-  (sin (degrees->radians-internal degrees)))
+  (if(= (modulo degrees 90) 0)
```

Comment:
Minor code style thing: I think generally it's accepted to put a space before the first `(` in a sequence of `(`, so there should be a space between the `if` and the `(` that comes after it, e.g., `(if (=`. I recommend this change for everywhere it occurs in your change.